x/liquidfarming/keeper/auction.go

```diff
@@ -109,112 +109,163 @@ func (k Keeper) RefundBid(ctx sdk.Context, auctionId uint64, poolId uint64, bidd
 	return nil
 }
 
-// getNextAuctionIdWithUpdate increments rewards auction id by one and store it.
-func (k Keeper) getNextAuctionIdWithUpdate(ctx sdk.Context, poolId uint64) uint64 {
-	id := k.GetLastRewardsAuctionId(ctx, poolId) + 1
-	k.SetLastRewardsAuctionId(ctx, id, poolId)
-	return id
-}
-
 // CreateRewardsAuction creates new rewards auction and store it.
-func (k Keeper) CreateRewardsAuction(ctx sdk.Context, poolId uint64, duration time.Duration) {
-	nextAuctionId := k.getNextAuctionIdWithUpdate(ctx, poolId)
-	poolCoinDenom := liquiditytypes.PoolCoinDenom(poolId)
-	startTime := ctx.BlockTime()
-	endTime := startTime.Add(duration)
+func (k Keeper) CreateRewardsAuction(ctx sdk.Context, poolId uint64, startTime, endTime time.Time) {
```

Comment:
As the function gets `endTime`, I thought it would be natural to get `startTime` as well, but it can be removed. Do you think it should be removed if the function doesn't necessarily need to accept the parameter?